libs/common-utils/src/tokens.ts

```diff
@@ -1,13 +1,18 @@
 import {
+  NATIVE_CURRENCIES,
   NATIVE_CURRENCY_ADDRESS,
   WRAPPED_NATIVE_CURRENCIES as WETH,
-  NATIVE_CURRENCIES,
 } from '@cowprotocol/common-const'
 import { SupportedChainId as ChainId } from '@cowprotocol/cow-sdk'
 
-
 export function isNativeAddress(tokenAddress: string, chainId: ChainId) {
-  return tokenAddress === 'ETH' || tokenAddress === NATIVE_CURRENCIES[chainId].symbol
+  if (!tokenAddress || !chainId) return false
+
+  if (tokenAddress === 'ETH') return true
```

Comment:
Should we match the pattern in doesTokenMatchSymbolOrAddress (libs/common-utils/src/doesTokenMatchSymbolOrAddress.ts) which uses toLowerCase() for making the address and symbol comparisons case-insensitive?

Reply:
https://github.com/cowprotocol/cowswap/pull/5658/commits/594d5177bf0de3891e32b66d3dfba1f027b184e7